src/domains/tasks/CreateTask/types.tsx

```diff
@@ -34,21 +34,33 @@ export const TaskTimeEstimateSchema = z.enum([
 
 export const CreateTaskFormSchema = z.object({
   category: TaskCategorySchema.default("Cleaning"),
-  title: z.string().min(10),
-  details: z.string().min(25),
-  budget: z.number().gt(5),
+  title: z
+    .string({
+      required_error: "Title is required",
+    })
+    .min(10, { message: "Title must be at least 10 characters long" }),
+  details: z
+    .string({ required_error: "Details are required" })
+    .min(25, { message: "Details must be at least 25 characters long" }),
+  budget: z.coerce
```

Comment:
Had to coerce `number` values as number inputs values are stored as strings by default.

Not sure if this is a Polaris thing.